### DISCUSSION TITLE
[AnalyticalTable] - Any way to detect Filter change?

### DISCUSSION THREAD

**Discussion by mkska** [UI5/webcomponents-react#5626] [Q&A] [answered]
Hi,

Using AnalyticalTables 1.26.0.
I'd like to [have manual sorting](https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-analyticaltable-recipes--docs#how-to-manually-sort-filter-or-group-the-table) on serverside; I want to trigger new data fetch based configured on filters once client made any changes on filters (column filters), but I couldn't find any callback.

How can I get a filter change event?

Thanks,
/mkska

**Comment by Lukas742** [comment, marked as answer]
Hi @mkska 

please open a [feature request](https://github.com/SAP/ui5-webcomponents-react/issues/new/choose) for the filter-change event, as currently you can only subscribe to internal state changes by e.g. leveraging useControlledState `reactTableOption`. [Here](https://stackblitz.com/edit/github-m1f4ja?file=src%2FApp.tsx) you can find an example of how an implementation could look like.
